test/common/testlib.py

```diff
@@ -1224,8 +1285,10 @@ def runOne(self, test, offset):
             result.printErrors()
             return result.wasSuccessful()
 
-    def run(self, testable):
-        tap.TapResult.plan(testable)
+    def run(self, testable, testable_no_collapse):
```

Comment:
What does "collapse" mean here? That could use a docstring (or comment)

Reply:
well the testable we flatten into a single list so they can be run concurrently, the latter we don't wanna  flatten. I'll add a comment, and maybe change the name to flatten